app/views/lks/copyright.html

```diff
@@ -35,6 +35,13 @@ <h2>Copyright Act & 'Library Privilege'</h2>
             <p>If copies cannot be made under CLA Licence Plus, NHS library staff may supply copies under terms of the <a href="#">Copyright, Designs and Patents Act of 1988</a>. This includes digital as well as printed works.</p>
 
             <p>See also The British Library’s help about <a href="#">Library Privilege.</a></p>
+
+            <figure class="nhsuk-image">
```

Comment:
Could we replace the HTML with it's corresponding nunjucks macro (e.g. https://service-manual.nhs.uk/design-system/components/images) ?

```
{{ image({
  "src": "https://assets.nhs.uk/prod/images/S_1017_allergic-conjunctivitis_M15.2e16d0ba.fill-320x213.jpg",
  "alt": "Picture of allergic conjunctivitis",
  "caption": "Itchy, red, watering eyes"
}) }}
```